JavaScript/TypeScript.sublime-syntax

```diff
@@ -726,7 +726,7 @@ contexts:
       scope: keyword.operator.type.js
 
     - match: as{{identifier_break}}
-      scope: keyword.operator.type.js
+      scope: keyword.operator.type.as.js
```

Comment:
The keyword `as` appears in many syntaxes with very similar meaning as well.

It is often used to assign the name or value from a left hand operator to the right.

As such it is just a `keyword.operator.assignment`. Maybe `keyword.operator.assignment.alias` as this is what it is often used for.